pkg/k8s/commands/run.go

```diff
@@ -73,13 +73,12 @@ func run(ctx context.Context, opt cmd.Option, cluster string, artifacts []*artif
 	if err != nil {
 		return xerrors.Errorf("k8s scan error: %w", err)
 	}
-
 	if err := report.Write(r, report.Option{
 		Format:     opt.Format,
 		Report:     opt.KubernetesOption.ReportFormat,
 		Output:     opt.Output,
 		Severities: opt.Severities,
-	}); err != nil {
+	}, opt.ReportOption); err != nil {
```

Comment:
It seems the only reason we are passing `opt.ReportOption` is because of `SecurityChecks`, but the report have its own option already, can we instead do:
```
f err := report.Write(r, report.Option{
		Format:     opt.Format,
		Report:     opt.KubernetesOption.ReportFormat,
		Output:     opt.Output,
		Severities: opt.Severities,
		SecurityChecks: opt.ReportOption.SecurityChecks,
	}); err != nil {
...
        }
```